GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift

```diff
@@ -179,8 +134,9 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> Void
      
      - returns: A view controller instance allowing the user to take a picture or pick a document.
      */
-    public init(giniConfiguration: GiniConfiguration) {
+    public init(giniConfiguration: GiniConfiguration, currentDevice: UIDevice = .current) {
```

Comment:
Is there a particular reason you chose to inject the device via the constructor? I'm not sure it would be better, but maybe just a settable property would be enough. As far as I understand it's only there because of testing.